### PR TITLE
Fix #358 Adding reference to ADB installation documentation

### LINKED ISSUE #358: Create link to ADB documentation for troubleshooting of Vagrant / Virutalization install

### PR DESCRIPTION
**Pull request by hferentschik**

### CODE REVIEW

**Comment by hferentschik** [comment]
@budhrg WDYT? Shall we merge this?

**Comment by brgnepal** [comment]
LGTM :+1:

File: README.adoc [hunk 1]
```diff
@@ -35,6 +35,10 @@ Vagrant plugin installation method:
 $ vagrant plugin install vagrant-service-manager
 ------------------------------------------------
 
+NOTE: If you are encountering problems with the plugin installation,
+refer to the https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst[ADB installation instructions]. There you can find
+more information on how to install Vagrant on various systems.
+
```
Comment (Preeticp) [inline README.adoc:40]: How about:To resolve any issues with the plugin installation and for more information on how to install Vagrant on various systems, refer to the ADB installation instructions.

**Comment by Preeticp** [comment]
Reference to Installation document already exists in the example execution of the Plugin. As such, do we need this here again?

File: README.adoc (hunk 1 above)
Comment (hferentschik) [inline README.adoc:40]: @Preeticp nice. Sounds better indeed. I'll make the change and merge :-)

**Pull request merged**